package.json

```diff
@@ -11,7 +11,8 @@
     "start": "webpack-dev-server -d --content-base ./client/public --config ./client/webpack.config.js",
     "start:server": "npm run build:watch",
     "build:server": "tsc",
-    "build:watch": "nodemon --exec ./node_modules/.bin/ts-node -- ./_server/index.ts"
+    "build:watch": "nodemon --exec ./node_modules/.bin/ts-node -- ./_server/index.ts",
+    "test": "jest"
```

Comment:
jest has command for: watch mode, for updating snapshots. Please check and create script entry for those as well.